pkg/build/builder/daemonless.go

```diff
@@ -305,32 +309,101 @@ func buildDaemonlessImage(sc types.SystemContext, store storage.Store, isolation
 
 func generateTransientMounts() []string {
 	mounts := []string{}
-	mounts = appendRHSMMount(mounts)
+	mounts = appendRHSMMount(defaultMountStart, mounts)
```

Comment:
it looks like all of these `append<something>mount` functions do a very similar thing. Can any of them be refactored into one function that takes parameters to control the processing? Or are they to different for that?

Reply:
yeah maybe appendETCPKIMount and appendRHSMMount

I'll give that a go and report back with an official yay/nay

the other two are different enough that it becomes untenable IMO

Reply:
yep worked out OK ... part of next push